### PR TITLE
Front-end mtg #8 Update variables in bootstraper

### PR DESCRIPTION
**Pull request by michalsnik**
- Add new files for variables
- Update imports

### CODE REVIEW

File: lib/netguru_bootstrapper/templates/application.scss [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+@import 'settings/color-variables';
 @import 'settings/z-index-variables';
-@import 'settings/variables';
+@import 'settings/typography-variables';
 @import 'framework/variables';
 
 @import 'framework/components';
 
 @import 'settings/breakpoint-variables';
+@import 'settings/components-variables';
```
Comment (czajkovsky) [inline lib/netguru_bootstrapper/templates/application.scss]: These two should be before `framework/variables`.

**Comment by czajkovsky** [comment]
@michalsnik thanks for the PR. Two things are missing:
1. Please update GEM wiki: https://github.com/netguru/netguru_bootstrapper#tree-structure
2. Please bump GEM version to `1.0.0` [here](https://github.com/netguru/netguru_bootstrapper/blob/master/lib/netguru_bootstrapper/version.rb) and add corresponding [changelog item](https://github.com/netguru/netguru_bootstrapper/blob/master/CHANGELOG.md).

**Comment by michalsnik** [comment]
Thanks @czajkovsky. PR updated

**Comment by czajkovsky** [comment]
Great job @michalsnik , thanks a lot!

**Pull request merged**